src/main/java/org/fogbowcloud/manager/core/models/orders/ComputeOrder.java

```diff
@@ -19,24 +19,37 @@
 	/** Disk attribute, must be set in GB. */
 	@Column(name = "disk")
 	private int disk;
-	
+
 	@Column(name = "image_name")
 	private String imageName;
 
 	@OneToOne(cascade = CascadeType.ALL)
 	@JoinColumn(name = "user_data_id")
 	private UserData userData;
 
-	public ComputeOrder(){ }
+	public ComputeOrder() {
+	}
 
-	public ComputeOrder(OrderState orderState, Token localToken, Token federationToken, String requestingMember,
-						String providingMember, OrderInstance orderInstance, long fulfilledTime, int vCPU, int memory,
-						int disk, String imageName, UserData userData) {
-		super(orderState, localToken, federationToken, requestingMember, providingMember, orderInstance, fulfilledTime);
+	/**
+	 * Creating Order with predefined Id.
+	 */
+	public ComputeOrder(String id, Token localToken, Token federationToken, String requestingMember, String providingMember,
+			int vCPU, int memory, int disk, String imageName, UserData userData) {
```

Comment:
same here. make a constructor call the other, if possible

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/f76e0578247c51bfa5f1fbb8168a77f6fa13c438).